playerUi/playbackControlsUIWithImaAds.html

```diff
@@ -0,0 +1,199 @@
+  <div class="container">
+    <h1>HTML5 Adaptive Streaming Player for MPEG-DASH & HLS</h1>
+    <h2>Your videos play everywhere with low startup delay, no buffering and in highest quality.</h2>
+    <div class="content">
+      <div class="player-wrapper">
+        <div id="player"></div>
+      </div>
+      <div class="description">
+        <p>For more information about the bitmovin player, please have a look at our online <a
+          href="//bitmovin.com/docs" target="_blank">Documentation</a>.</p>
+      </div>
+    </div>
+  </div>
+</div>
+
+<script type="text/javascript">
+  var conf = {
+    key: "YOUR-PLAYER-LICENSE-KEY",
+    advertising: {
+        adBreaks: [
+        {
+            tag: {
+              url:
+              'https://pubads.g.doubleclick.net/gampad/ads?sz=640x480&iu=/124319096/external/ad_rule_samples&ciu_szs=300x250&ad_rule=1&impl=s&gdfp_req=1&env=vp&output=vmap&unviewed_position_start=1&cust_params=deployment%3Ddevsite%26sample_ar%3Dpremidpostpod&cmsid=496&vid=short_onecue&correlator=',
+              type: 'vmap'
+            },
+        }
+        ],
+    },
+    playback: {
+      autoplay: true,
+      muted:true
+    },
+  };
+
+  var player = new bitmovin.player.Player(document.getElementById('player'), conf);
+
+  // UI for IMA SDK
+  // Create a separate UI manager that shows a custom UI for all ads
+  var imaUIManager= new bitmovin.playerui.UIManager(player, [{
+    ui: new bitmovin.playerui.UIContainer({
+      userInteractionEventSource: document.getElementById('player'), // show controls on user interaction
+      //hideDelay: -1, // always show UI
+      hideDelay: 3000, // hide controls in 3 second
+      components: [
+        new bitmovin.playerui.Container({
+        }),
+        new bitmovin.playerui.ControlBar({
+          components: [
+            new bitmovin.playerui.Container({
+              components: [
+                playbackButton = new bitmovin.playerui.PlaybackToggleButton(),
+                new bitmovin.playerui.VolumeToggleButton(),
+                new bitmovin.playerui.VolumeSlider(),
+              ],
+              cssClasses: ['controlbar-bottom'],
+            }),
+          ],
+        cssClasses: ['adControlBar']
+      }),
+      ],
+      cssClasses: ['ui-skin-ads']
+    }),
+    condition: (context) => {
+      // Show the ads UI for every ad by ignoring the context.adRequiresUi flag
+      return context.isAd;
+    },
+  }], {metadata: {}});
+
+  var sourceConfig = {
+    title: "Bitmovin Player",
+    description: "This demo showcases the possibility to add playback controls UI during Ad playback!",
+    dash: "https://bitmovin-a.akamaihd.net/content/MI201109210084_1/mpds/f08e80da-bf1d-4e3d-8899-f0f6155f6efa.mpd",
+    hls: "https://bitmovin-a.akamaihd.net/content/MI201109210084_1/m3u8s/f08e80da-bf1d-4e3d-8899-f0f6155f6efa.m3u8",
+    progressive: "https://bitmovin-a.akamaihd.net/content/MI201109210084_1/MI201109210084_mpeg-4_hd_high_1080p25_10mbits.mp4",
```

Comment:
Inconsistent usage of double and single quotes - I'd suggest using single quotes

Reply:
updated as suggested